packages/story-editor/src/components/tabview/index.js

```diff
@@ -259,12 +260,13 @@ function TabView({
       {tabs.map(({ id, title, icon: Icon, ...tabRest }) => (
         <Tab
           key={id}
-          ref={(tabRef) => {
+          ref={(node) => {
             if (tabRefs) {
-              tabRefs[id].current = tabRef;
+              tabRefs[id].current = node;
             } else {
-              internalTabRefs.current[id] = tabRef;
+              internalTabRefs.current[id] = node;
             }
+            onTabRefUpdated(node, id);
           }}
```

Comment:
I like this pattern. It makes sense to keep references to the element internally and externally so we can interact with the element rather than deciding which ref to set based on what's passed in to the components.